### ISSUE TITLE
[Package Request]: Virtual Audio Cable Lite

### ISSUE THREAD

**Issue by AsciiWolf** [microsoft/winget-pkgs#160589]
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://vac.muzychenko.net/en/download.htm
Publisher: Eugene Muzychenko
Package Name: Virtual Audio Cable Lite
Description: Audio bridge between applications that transmits sounds from app to app, from device to device.
Package Version: 4.70
Installer URL: https://software.muzychenko.net/freeware/vac470lite.zip

**Comment by Trenly** [comment]
It looks like the installer is a 7z SFX. Do you know the silent switches required for this package to install unattended?

**Comment by AsciiWolf** [comment]
I am not sure, but `-y -gm2` may work.

**Comment by Dragon1573** [comment]
> I am not sure, but `-y -gm2` may work.

Tried and not valid. The `*.chm` manual in it does not mention about silent installation.

![image](https://github.com/microsoft/winget-pkgs/assets/49941141/d9119fb7-e69b-49bb-89e6-5964eecb6906)

**Comment by AsciiWolf** [comment]
@Dragon1573 Please try the new Installer URL. It is a regular zip file with setup.exe inside that should support silent installation by using `-i -h` switches.

**Comment by Dragon1573** [comment]
Nope ... Switches still invalid.

```text
22:18:12 D:\...\Compressed 2ms pwsh> Invoke-WebRequest -OutFile vac470lite.zip 'https://software.muzychenko.net/freeware/vac470lite.zip'
22:18:59 D:\...\Compressed 4.557s pwsh> ls .\vac470lite.zip

    Directory: D:\Download\Aria2\Compressed

Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
-a---           2024/6/29    22:18        1340468 vac470lite.zip

22:19:10 D:\...\Compressed 14ms pwsh> D:\Applications\Git\usr\bin\unzip.exe .\vac470lite.zip
Archive:  .\vac470lite.zip
   creating: arm64/
  inflating: arm64/audiorepeater.exe
  inflating: arm64/audiorepeater_ks.exe
  inflating: arm64/vcctlpan.exe
  inflating: arm64/vrtaucbl.sys
 extracting: homepage.url
  inflating: license.txt
  inflating: readme.txt
  inflating: setup.exe
  inflating: setup.ini
  inflating: setup64.exe
  inflating: setup64a.exe
   creating: tools/
  inflating: tools/delete_phantom.cmd
  inflating: tools/delete_service.cmd
  inflating: tools/delete_settings.cmd
 extracting: tools/open_audio_properties.cmd
 extracting: tools/open_device_manager.cmd
  inflating: tools/setregdword.cmd
  inflating: tools/setvars.cmd
  inflating: tools/set_audiodg_instance_count_report.cmd
  inflating: tools/set_cable_signal_level_disable.cmd
  inflating: tools/set_check_level.cmd
  inflating: tools/set_connected_source_lines_mask.cmd
  inflating: tools/set_debug_level.cmd
  inflating: tools/set_format_attribute_policy.cmd
  inflating: tools/set_hiding_mode.cmd
  inflating: tools/set_packet_mode.cmd
  inflating: tools/set_portcls_policy.cmd
  inflating: tools/set_port_type.cmd
  inflating: tools/set_restrict_affinity.cmd
  inflating: tools/set_restrict_client_affinity.cmd
  inflating: tools/set_restrict_kmixer_affinity.cmd
  inflating: tools/set_source_line_types_mask.cmd
  inflating: tools/set_split_dpc_processing.cmd
  inflating: tools/set_stream_signal_level_disable.cmd
  inflating: tools/set_timer_precision_percent.cmd
  inflating: tools/set_timer_resolution_percent.cmd
  inflating: vac.chm
  inflating: vrtaucbl.cat
  inflating: vrtaucbl.inf
  inflating: vrtaucbl6x.cat
  inflating: vrtaucbl6x.inf
   creating: x64/
  inflating: x64/audiorepeater.exe
  inflating: x64/audiorepeater_ks.exe
  inflating: x64/vcctlpan.exe
  inflating: x64/vrtaucbl.sys
   creating: x86/
  inflating: x86/audiorepeater.exe
  inflating: x86/audiorepeater_ks.exe
  inflating: x86/vcctlpan.exe
  inflating: x86/vrtaucbl.sys
22:20:05 D:\...\Compressed 236ms pwsh> .\setup.exe -i -h

# Here jumps out UAC request, approve it with "YES" button.
```

![image](https://github.com/microsoft/winget-pkgs/assets/49941141/254d14fb-a885-4e0e-b97e-77cde597b04b)

**Comment by AsciiWolf** [comment]
You can also try `-s`.

**Comment by AsciiWolf** [comment]
This is the e-mail that I got from VAC developer:

>Silent installation option is provided for license packs starting from 50 units.

So, sadly, although there are [some hacks](https://stackoverflow.com/questions/58463743/is-there-any-way-to-install-virtual-audio-cable-application-without-prompt-messa) to get it working, it is not officially supported except for the enterprise installers that are not publicly available. :-/

Closing this issue. Will re-open if anything changes in the future.
